packages/expo-video/src/VideoView.types.ts

```diff
@@ -0,0 +1,13 @@
+import { VideoPlayer } from './VideoPlayer';
+
+type VideoContentFit = 'contain' | 'cover' | 'fill';
+
+export type VideoViewProps = {
+  player: VideoPlayer;
+  nativeControls: boolean | undefined;
+  contentFit: VideoContentFit | undefined;
+  //   contentPosition: CGVector | undefined;
+  allowsFullscreen: boolean | undefined;
+  showsTimecodes: boolean | undefined;
+  requiresLinearPlayback: boolean | undefined;
```

Comment:
Would be nice to add doc comments already, at least with the `@platform` tag so we can clearly see what is implemented on specific platform